scripts/environment/bootstrap-ubuntu-20.04.sh

```diff
@@ -3,7 +3,11 @@ set -e -o verbose
 
 export DEBIAN_FRONTEND=noninteractive
 
+apt-get install -y grub-efi
+update-grub
+
 apt update -y -qq
+apt upgrade -y -qq
```

Comment:
Did you end up needing this? I see it wasn't mentioned in https://github.com/actions/virtual-environments/issues/1605

Reply:
Yeah - that's the original step that was failing. The grub-efi packages was dying during the apt upgrade.